src/tools/PICS-generator/XMLPICSValidator.py

```diff
@@ -0,0 +1,122 @@
+#
```

Comment:
`README.md` in this folder seems to describe the content of the file. We should add information about this new script there.

Reply:
Sure, I can do that. It was mostly intended for myself to catch these naming issues between PICS and DM cluster names, but good point. I will add some text describing what the purpose of the new script is and how to use it.